setup.py

```diff
@@ -31,3 +33,5 @@
                                        'evalml/tests/data/churn.csv',
                                        'evalml/tests/data/fraud_transactions.csv.gz'])],
 )
+
+call("pip uninstall pystan -y", shell=True)
```

Comment:
Uninstall `PyStan` to avoid licensing issues by mistake

Reply:
I think this line belongs in `cmdstan-builder` as opposed to EvalML.

I commented out this line and added these lines:
```python
try:
    import pystan
    assert False, "pystan installed!"
except ModuleNotFoundError:
    print("pystan not installed!")
```

When I run
`python setup.py bdist_wheel` I see "pystan not installed!". Doing `pip freeze` verifies this. 

When I run `pip install dist/evalml-0.29.0-py3-none-any.whl`, `pystan` is also not installed.

However, when I run `pip install dist/evalml-0.29.0-py3-none-any.whl['prophet']`, pystan is installed!

![image](https://user-images.githubusercontent.com/41651716/128044393-4f8af78f-9611-4fbf-87f5-8548cc449f50.png)